app/jinja_filters.py

```diff
@@ -311,6 +302,30 @@ def format_date_range_no_repeated_month_year(context, start_date, end_date, date
     return mark_safe(context, output)
 
 
+def _get_first_valid_item(items):
+    """
+    :param items: anything that is iterable
+    :return: first ‘truthy’ value
+
+    Note: to guarantee the returned element is actually the first truthy element in the iterable,
+    'items' must be a data structure that preserves order, ie tuple, list etc.
+    If order is not important, this can be reused to return `one of` the elements which is truthy.
+    """
+    return next((item for item in items if item), None)
+
+
+@evalcontextfunction
+def display_first_valid_item(context, *items):
```

Comment:
This could be `display_first_truthy_item`. IMO it conveys more information.

Reply:
Can change, just thought it was too specific to be an acceptable name.

Reply:
A doc string does work, but still means you'd have to look up the filter to ensure you knew what it does.

I don't think there's any reason not to be specific in the name of a filter? It will only ever be used in a jinja2 template, which was designed for Python, so `truthy` isn't out of place I don't think.

Reply:
Maybe someone might use this on an array of numbers for instance, in which case:

input of `[0, 0.1, 0.2]`

would display 0.1, even though `0` is 'valid' to me.

Reply:
Good point. ^